src/index.ts

```diff
@@ -379,6 +390,8 @@ export function apply(ctx: Context, config: Config) {
           const res = await ctx.http.axios(trimSlash(config.endpoint) + path, {
             method: 'POST',
             timeout: config.requestTimeout,
+            // Since novelai's latest interface returns an application/x-zip-compressed, a responseType must be passed in
+            responseType: getResponseType(),
```

Comment:
Since there is only one exception here, you could just use the conditional expression here:

```

```suggestion
            responseType: ['login', 'token'].include(config.type) ? 'arraybuffer' : 'json',
```

```

Reply:
This is a neat and elegant suggestion, but I think writing a method will be more maintainable in the future.

Reply:
True, but in my thought this part would never goes complicated since there are only `json` or `arrabuffer` that we could handle here.